sdks/go/pkg/beam/xlang.go

```diff
@@ -173,6 +212,7 @@ func TryCrossLanguage(
 	expansionAddr string,
 	namedInputs map[string]PCollection,
 	namedOutputTypes map[string]FullType,
+	classpath []string,
```

Comment:
This is a breaking change we shouldn't make for a one off. If anything, we should break it harder than this so we can avoid breaking it in the future.

Reply:
I guess building the expansion address handler can avoid doing this. Got it.